spec/granite/comparisons/comparisons_spec.cr

```diff
@@ -0,0 +1,116 @@
+require "../../spec_helper"
+
+{% for adapter in GraniteExample::ADAPTERS %}
+module {{adapter.capitalize.id}}
```

Comment:
I really appreciate you following suite with the tests here, @astupidnerd! 

There is a pending pull which simplifies this syntax greatly and runs tests across different adapters in docker/CI instead of requiring devs to have all the adapters installed and configured to run tests.

Reply:
I'm happy to update the tests if need be. :)